src/ansys_sphinx_theme/search/fuse_search.py

```diff
@@ -68,6 +87,31 @@ def build_sections(self):
                 }
             )
 
+            self._process_desc_element(node, section_title)
+
+    def _process_desc_element(self, node, title):
+        """Process `desc` element within a section."""
+        for element in node.traverse(Element):
+            if element.tagname != "desc":
+                continue
+
+            # id is the id tag of the desc element
+            section_anchor_id = element.attributes["ids"]
+            if element.children:
+                for element_child in element.children:
+                    if element_child.tagname != "desc_signature":
+                        continue
+                    section_anchor_id = element_child.attributes["ids"][0]
+            section_text = element.astext()
+            section_title = f"{title} > {section_anchor_id}"
+            self.sections.append(
+                {
+                    "title": section_title,
+                    "text": section_text,
+                    "anchor_id": section_anchor_id,
```

Comment:
This is working fine. However, the only anchors containing a "." are the ones generated by the Python domain. Even if users create a `The CONTRIBUTING.md anchor`, the "." gets replaced automatically by a "-".

Knowing this, would it be possible to split the anchor using as character the "."? If so, then we should keep the latest item, which is the actual method or property.

This leads to a nice breadcrumb like `FooClass > Methods > foo_method` instead of `FooClass > Methods > FooClass.foo_method`.

Reply:
i agree, let me try to get that.

Reply:
[89081f3](https://github.com/ansys/ansys-sphinx-theme/pull/541/commits/89081f394fe644e0055c455e5a34ebf6360883ac) added this